app/models/foreman_virt_who_configure/output_generator.rb

```diff
@@ -9,7 +9,7 @@ def initialize(code, identifier, message)
       end
     end
 
-    MINIMUM_VIRT_WHO_VERSION = '0.19'
+    MINIMUM_VIRT_WHO_VERSION = '0.22.2'
```

Comment:
0.22.5 even seem to no support the kubeconfig configuration option

Reply:
This https://github.com/candlepin/virt-who/pull/188 provides support for this option. Once it is merged we need to update version here and it should work correctly.

Reply:
this has been merged but there's no release yet